README.md

```diff
@@ -262,6 +262,29 @@ The query will be performed on the field if the condition is satisfied. The foll
     }
     ```
 
+## Dump and Load
+
+By default, the hnsw index is in memory. You can dump the index to `data_path` by calling `.dump()`:
+
+```python
+
+from annlite import AnnLite
+
+ann = AnnLite(128, metric='cosine', data_path="/path/to/data_path")
+ann.index(docs)
+ann.dump()
+ann.close()
+```
+Note that please make sure to call `.close()` before you restore the index.
```

Comment:
there is no need to `.close()` before restoring from local because it's jsut load indexer file to hnsw and sqlite. And for restoring from remote, the close logic is inside the restore function.

Reply:
I see.